crawl-ref/source/mon-info.cc

```diff
@@ -1825,6 +1783,502 @@ void get_monster_info(vector<monster_info>& mons)
     sort(mons.begin(), mons.end(), monster_info::less_than_wrapper);
 }
 
+void mons_to_string_pane(string& desc, int& desc_colour, bool fullname, 
+                         const vector<monster_info>& mi, int start, 
+                         int count)
+{
+    mi[start].to_string(count, desc, desc_colour, fullname, nullptr, false);
+    mons_conditions_string(desc, mi, start, count, true);
+}
+
+void mons_conditions_string(string& desc, const vector<monster_info>& mi, 
+                            int start, int count, bool listall)
+{
+    static vector<monster_info_func> monster_info_funcs = 
+    init_monster_info_funcs();
+    
+    ostringstream out;
+    if (listall)
+        out << desc;
+    bool first=true; bool didany = false;
+    for (auto &info: monster_info_funcs) {
+        int num = 0;
+        for (int j = start; j < start+count; j++)
+        {
+            if (info.test(mi[j], listall))
+            {
+                if (!listall)
+                {
+                    out << info.singular;
+                    desc=out.str();
+                    return;
+                }
+                else
+                {
```

Comment:
crawl code style for single line blocks like here is to skip the braces. `util/unbrace` can automatically fix up all instances of this, but I don't remember the exact command

Reply:
I'm not sure if you mean "please resubmit making this change" (which I guess I'll have to figure out how to do) or "just FYI..." or "here's a note that if we accept this we'll have to change that".

Reply:
It will help speed up an eventual acceptance if you make the change.

You can continue to push commits to your branch, GitHub will add them to this PR automatically. You can also force-push your branch (if you need to rebase &c.)

Reply:
Done, I think. (I noticed util/unbrace unbraced something of the form if (foo) \n { \n this; that; the_other; } from which I infer that you'd prefer not to have multiple statements on one line like that).

Reply:
yeah, it's not explicitly part of the coding conventions but we do avoid that in all but a very few specific cases (e.g. `case X: something; break;`).